3-networks-dual-svpc/envs/development/variables.tf

```diff
@@ -29,6 +29,16 @@ variable "terraform_service_account" {
   description = "Service account email of the account to be added to the VPC-SC perimeter."
 }
 
+variable "projects_service_account" {
```

Comment:
Lets do the remote state approach we discussed.

Reply:
now reading the service accounts from remote state